agent/dockerclient/dockerapi/docker_client_integ_test.go

```diff
@@ -52,6 +54,9 @@ func TestImageManifestPullInteg(t *testing.T) {
 	if err != nil {
 		t.Skipf("Skipping test due to unsupported Docker version: %v", err)
 	}
+	// Make retries very fast
+	supportedClient.(*dockerGoClient).manifestPullBackoff = retry.NewExponentialBackoff(
```

Comment:
Just curious, how long do manifest pulls usually take? Will there be a risk of flakiness here?